bin/start

```diff
@@ -31,9 +59,13 @@ if [ ! -f "$STARTED_ONCE" ]; then
     # touch $STARTED_ONCE
 fi
 
+if [ -z "$TAIL" ]; then
+    COMPOSE_UP_ARGUMENTS="-d"
+fi
+
 # them spool them up, this is in an effort to reduce what is happening
 # simultaneously and perhaps provide a more consistent startup experience
-docker-compose up -d 2>&1 | filterUp
+docker-compose up $COMPOSE_UP_ARGUMENTS 2>&1 | filterUp
```

Comment:
Does running this with tail mean you don't get the final ("It may take a moment..") message until after you kill the process?  Seems that's a tiny bug.

Reply:
Right. Any suggestions how to most elegantly handle that?

Reply:
I think we can change the messages so they say:
"Starting stack with website running at http://localhost:3020..."

Then depending on this flag, either follow, or the ping website until it's ready then say taht.